cmake/gcc-gcov.cmake

```diff
@@ -17,7 +17,7 @@ find_package(Find REQUIRED)
 set(GCOV_OPTS "-pb")
 
 if(NOT APPLE)
-  # DONT elide source prefix.
+  # DON'T elide source prefix.
```

Comment:
`elide` ?

Reply:
@LebedevRI please advise..... should it be `DON'T elude source prefix` ?

Reply:
It's talking about https://gcc.gnu.org/onlinedocs/gcc/Invoking-Gcov.html
> * -r
    --relative-only
    Only output information about source files with a relative pathname (after source prefix **elision**). Absolute paths are usually system header files and coverage of any inline functions therein is normally uninteresting.

I believe the current variant is the proper one.